packages/commonwealth/server/controllers/server_threads_methods/update_thread.ts

```diff
@@ -102,6 +102,9 @@ export async function __updateThread(
   if (discordMeta) {
     threadWhere.discord_meta = discordMeta;
   }
+  if (!body) {
```

Comment:
`body` is optional in the `UpdateThreadOptions` type above. Either the type is wrong or the route is allowed to be called without a body.